playbooks/roles/journals/defaults/main.yml

```
@@ -166,6 +166,7 @@ JOURNALS_CORS_ORIGIN_WHITELIST: []
 JOURNALS_ALLOW_CORS_HEADERS: true
 JOURNALS_ALLOW_CORS_CREDENTIALS: true
 
+JOURNALS_API_ENABLE_NEWRELIC_DISTRIBUTED_TRACING: false
```

Perhaps this shouldn't start with ```JOURNALS_API``` since none of the other JOURNALS variables start with ```JOURNALS_API```
```suggestion
JOURNALS_ENABLE_NEWRELIC_DISTRIBUTED_TRACING: false
```